test/Microsoft.Azure.WebJobs.Host.UnitTests/PublicSurfaceTests.cs

```diff
@@ -126,7 +126,9 @@ public void WebJobsPublicSurface_LimitedToSpecificTypes()
                 "StorageAccountAttribute",
                 "DisableAttribute",
                 "TimeoutAttribute",
-                "TraceLevelAttribute"
+                "TraceLevelAttribute",
+                "IResolutionPolicy",
+                "ResolutionContext"
```

Comment:
yeah, i didn't fix those tests -- figured i'd wait till it was all ready (b/c the names kept changing anyway)